From b2a49ef2285244f75e258eb7de4bedb6b988203e Mon Sep 17 00:00:00 2001
From: Andrei Pistirica <sorin.pistirica@freescale.com>
Date: Fri, 11 May 2012 11:29:12 +0300
Subject: [PATCH 029/162] FMD16 candidate 1 compile in p5020 64bit. Add
 comment

Adding missing compat structures.
Changed from error to warning where the IOCTL are not available.
fman_test.c had an issue (workaround only).
Conflict edited.

Signed-off-by: Andrei Pistirica <sorin.pistirica@freescale.com>
[Kevin: Original patch taken from FSL
QorIQ-SDK-V1.3-SOURCE-20121114-yocto.iso tarball.]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 .../dpa/NetCommSw/src/wrapper/fman_test.c          |    1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/freescale/dpa/NetCommSw/src/wrapper/fman_test.c b/drivers/net/ethernet/freescale/dpa/NetCommSw/src/wrapper/fman_test.c
index 6238ef4..9628eab 100644
--- a/drivers/net/ethernet/freescale/dpa/NetCommSw/src/wrapper/fman_test.c
+++ b/drivers/net/ethernet/freescale/dpa/NetCommSw/src/wrapper/fman_test.c
@@ -783,6 +783,7 @@ static void fmt_pcd_fqs_dqrr(
 }
 
 /* private pcd queue template */
+//static const struct qman_fq pcd_fq __devinitconst = {
 static const struct qman_fq pcd_fq = {
 	.cb = { .dqrr = fmt_pcd_dqrr,
 		.ern = fmt_pcd_err_dqrr,
-- 
1.7.9.7

